### PR TITLE
use allocated instead of settled

### PR DESCRIPTION
**Pull request by JessicaGreben**
What?
Change the project resource limiting to limit on the Orders.allocated instead of Orders.settled for bandwidth.  Also add an expansion factor to the limit so that uplinks can use an approx raw 25GB.

Why?
We want to limit on Orders `allocated`, not `settled`, otherwise uplinks and storage nodes could behave poorly and potentially upload as much data as they want.

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Could the PR be broken into smaller PRs?
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?

### CODE REVIEW

File: pkg/accounting/projectusage_test.go [hunk 1]
```diff
@@ -30,69 +30,118 @@ func TestProjectUsage(t *testing.T) {
...
 					require.NoError(t, err)
 				}
 
+				// Execute test: get storage totals for a project, then check if that exceeds the max usage limit
+				inlineTotal, remoteTotal, err := acctDB.ProjectStorageTotals(ctx, projectID)
+				require.NoError(t, err)
+				maxAlphaUsage := 25 * memory.GB
+				actualExceeded, actualResource := accounting.ExceedsAlphaUsage(0, inlineTotal, remoteTotal, maxAlphaUsage)
+				require.Equal(t, tt.expectedExceeded, actualExceeded)
+				require.Equal(t, tt.expectedResource, actualResource)
```
Comment (mniewrzal) [inline pkg/accounting/projectusage_test.go]: doubled Equal

File: pkg/accounting/projectusage_test.go [hunk 2]
```diff
@@ -30,69 +30,118 @@ func TestProjectUsage(t *testing.T) {
...
 					require.NoError(t, err)
 				}
 
-				// Execute test: get storage and bandwidth totals for a project, then check if that exceeds the max usage limit
-				inlineTotal, remoteTotal, err := acctDB.ProjectStorageTotals(ctx, projectID)
+				// Setup: create some bytes for the uplink to upload to test the download later
+				expectedData := make([]byte, 50*memory.KiB)
+				_, err = rand.Read(expectedData)
 				require.NoError(t, err)
+				actualErr := planet.Uplinks[0].Upload(ctx, planet.Satellites[0], bucketName, "test/path", expectedData)
```
Comment (mniewrzal) [inline pkg/accounting/projectusage_test.go]: check `actualErr`

File: satellite/satellitedb/accounting.go [hunk 3]
```diff
@@ -28,7 +28,7 @@ func (db *accountingDB) ProjectBandwidthTotal(ctx context.Context, bucketID []by
 	pathEl := bytes.Split(bucketID, []byte("/"))
 	_, projectID := pathEl[1], pathEl[0]
 	var sum *int64
-	query := `SELECT SUM(settled) FROM bucket_bandwidth_rollups WHERE project_id = ? AND action = ? AND interval_start > ?;`
+	query := `SELECT SUM(allocated) FROM bucket_bandwidth_rollups WHERE project_id = ? AND action = ? AND interval_start > ?;`
```
Comment (mniewrzal) [inline satellite/satellitedb/accounting.go]: Maybe we should rename method to `ProjectAllocatedBandwidthTotal`?

File: pkg/accounting/projectusage_test.go [hunk 4]
```diff
@@ -30,69 +30,118 @@ func TestProjectUsage(t *testing.T) {
...
 				if tt.expectedResource == "storage" {
 					now := time.Now()
 					err := setUpCreateTallies(ctx, projectID, acctDB, now)
 					require.NoError(t, err)
 				}
 
+				// Execute test: get storage totals for a project, then check if that exceeds the max usage limit
+				inlineTotal, remoteTotal, err := acctDB.ProjectStorageTotals(ctx, projectID)
+				require.NoError(t, err)
+				maxAlphaUsage := 25 * memory.GB
```
Comment (mniewrzal) [inline pkg/accounting/projectusage_test.go]: Why using `accounting.ExpansionFactor` in bandwidth test and not here?

Comment (JessicaGreben) [reply pkg/accounting/projectusage_test.go]: oops let me fix that

File: pkg/accounting/projectusage_test.go (hunk 1 above)
Comment (JessicaGreben) [reply pkg/accounting/projectusage_test.go]: what so you mean?

Comment (mniewrzal) [reply pkg/accounting/projectusage_test.go]: sorry to fast review, it looked identical :)

**Pull request merged**